.github/workflows/tests-skip.yml

```
@@ -45,4 +45,4 @@ jobs:
     if: always()
     runs-on: ubuntu-latest
     steps:
-      - run: 'echo "Nothing to analyze"'
+      - run: 'echo "No Source Changes and Skipping `test.yml`"'
```

```suggestion
      - run: 'echo "No source changes and skipping `test.yml`"'
```

Let's use title case here 😉

---

Perfect.. I'll change that..